microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -79,6 +81,9 @@ def run_tests(
                 r.status,
             )
 
+        self._save_dmesg_logs(log_path)
+
+        result.assert_exit_code()
         assert_that(failures, f"Unexpected failures: {failures}").is_empty()
```

Comment:
This would be unreachable statement as there is assert right above it and when there are failed subtests the exit code is not 0 from CH cmd.

Reply:
Fixed